src/nwb2bids/_command_line_interface/_main.py

```diff
@@ -112,23 +113,60 @@ def _run_convert_nwb_dataset(
 
     converter = convert_nwb_dataset(nwb_paths=handled_nwb_paths, run_config=run_config)
 
+    if silent:
+        return
+
     notifications = converter.messages
+    notifications_by_severity: dict[Severity, list[InspectionResult]] = collections.defaultdict(list)
+    for notification in notifications:
+        notifications_by_severity[notification.severity].append(notification)
+    errors = notifications_by_severity[Severity.ERROR]
+    criticals = notifications_by_severity[Severity.CRITICAL]
+
     console_notification = ""
-    if notifications:
-        notification_text = (
-            f'\n{(n:=len(notifications))} {_pluralize(n=n, word="suggestion")} for improvement '
-            f'{_pluralize(n=n, word="was", plural="were")} found during conversion.'
-        )
-        console_notification += rich_click.style(text=notification_text, fg="yellow")
+    if errors:
+        number_of_errors = len(errors)
+
+        top_three = errors[:3]
+        number_to_print = len(top_three)
 
-    if console_notification != "" and not silent:
+        text = (
+            "\nBIDS dataset was not successfully created!\n"
+            f'{_pluralize(n=number_to_print, phrase="An error was", plural="Some errors were")} '
+            "encountered during conversion.\n"
+        )
+        if number_to_print > 1 and number_of_errors > 3:
```

Comment:
```suggestion
        if number_of_errors > 3:
```

`number_to_print > 1` is `True` whenever `number_of_errors > 3` is `True`